<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QMainWindow" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>home</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>821</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>control</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>211</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Set handset ID</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>156</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>271</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Poll control</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop voting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Open voting</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>151</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Vote options</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>103</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="afsingle">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>A-F single</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="afmulti">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>A-F multi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="oneto100">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>1-100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="oneto100Avg">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>average</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>301</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>communication log</string>
     </property>
     <widget class="QTextEdit" name="communicationLog">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>190</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Reconnect</string>
     </property>
    </widget>
    <widget class="QLabel" name="countDown">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>571</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="resourceMonitor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Device resource monitor</string>
    </property>
    <widget class="QLabel" name="graph">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>801</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="connectionStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>791</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
